### PR TITLE
Basic text support for dynamic type

### PR DESCRIPTION
**Pull request by PavelHolec**
image

### CODE REVIEW

File: Sources/Orbit/Components/Heading.swift [hunk 1]
```diff
@@ -94,6 +94,34 @@ public extension Heading {
...
+        var textStyle: Font.TextStyle {
+            switch self {
+                case .display:          return .largeTitle
+                case .displaySubtitle:
+                    if #available(iOS 14.0, *) {
+                        return .title2
+                    } else {
+                        return .headline
+                    }
+                case .title1:           return .title
```
Comment (sjavora) [inline Sources/Orbit/Components/Heading.swift:107]: I'd probably put these on separate lines to match the cases with availability ifs... this is harder to read IMO.

File: Sources/Orbit/Foundation/Typography/Font.swift [hunk 2]
```diff
@@ -24,12 +26,12 @@ public extension Font {
...
 
-        return .custom(fontName, size: size)
+        return customFont(fontName, size: size, style: style)
     }
 
     /// Creates Orbit icon font.
-    static func orbitIcon(size: CGFloat) -> Font {
-        .custom("orbit-icons", size: size)
+    static func orbitIcon(size: CGFloat, style: Font.TextStyle = .body) -> Font {
+        return customFont(Self.orbitIconFontName, size: size, style: style)
```
Comment (sjavora) [inline Sources/Orbit/Foundation/Typography/Font.swift:34]: Why the `return`?

File: Sources/Orbit/Components/Heading.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Heading.swift]: Updated

File: Sources/Orbit/Foundation/Typography/Font.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Foundation/Typography/Font.swift]: Updated

**Pull request merged**